### PR TITLE
chore: update data loading in Keras CIFAR10 example

### PR DESCRIPTION
**Pull request by coreystatendet**
## Description

As requested by Ryan in [MLG-690].

Updates the example data loading to no longer use `ImageDataGenerator`.  Also uses Keras' built in for CIFAR10 download instead of our own.

## Test Plan

Manually run the `const.yaml` and `distributed.yaml` examples and confirm they succeed.

## Commentary (optional)

I didn't notice anything else clearly deprecated.

## Checklist

- [X] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [X] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

MLG-690

[MLG-690]: https://hpe-aiatscale.atlassian.net/browse/MLG-690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: examples/computer_vision/cifar10_tf_keras/const.yaml [hunk 1]
```diff
@@ -19,4 +17,5 @@ searcher:
     epochs: 32
 min_validation_period:
   epochs: 1
+max_restarts: 0
```
Comment (rb-determined-ai) [inline examples/computer_vision/cifar10_tf_keras/const.yaml:20]: while I do this on every example _I_ run, I don't think we should be setting this in our examples.

Though I'm not opposed to making it the default... I've always found this deeply annoying.

File: examples/computer_vision/cifar10_tf_keras/data.py [hunk 2]
```diff
@@ -1,95 +1,25 @@
...
-    return train
-
-
-def get_validation_data(data_directory):
-    (_, _), (test_data, test_labels) = get_data(data_directory)
-    return preprocess_data(test_data), preprocess_labels(test_labels)
+import tensorflow as tf
+
+
+def load_numpy_data(
```
Comment (rb-determined-ai) [inline examples/computer_vision/cifar10_tf_keras/data.py:8]: can we just put this in the model def?  I don't see why it deserves its own file anymore.

**Review by rb-determined-ai — APPROVED** [review]
thank you!

File: examples/computer_vision/cifar10_tf_keras/const.yaml (hunk 1 above)
Comment (coreystatendet) [reply examples/computer_vision/cifar10_tf_keras/const.yaml]: Removed.  I also do this on every example I run, so I do think it should be the default.

**Pull request merged**